Tests/SonarScanner.MSBuild.Shim.Test/PropertiesFileGeneratorTests.cs

```diff
@@ -39,7 +39,40 @@ public class PropertiesFileGeneratorTests
         #region Tests
 
         [TestMethod]
-        public void FileGen_NoProjectInfoFiles()
+        public void PropertiesFileGenerator_WhenConfigIsNull_Throws()
```

Comment:
These argument throwing tests could be single line ones just like [this](https://github.com/SonarSource/sonar-scanner-msbuild/blob/05499245d82522b5543db07355cc20b10631335a/Tests/SonarScanner.MSBuild.Test/AnalysisWarning/WarningsSerializerTest.cs#L36) one.